llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -4214,9 +4214,15 @@ VectorizationFactor LoopVectorizationPlanner::selectVectorizationFactor() {
                 }
               }
             }
-            [[fallthrough]];
```

Comment:
Hmm, I don't think this is right because previously when falling through it was hitting the VPInstruction::ExplicitVectorLength case below and adding on the cost of the instruction, i.e. `C += VPI->cost(VF, CostCtx)`.

If you change this to

```
  C += VPI->cost(VF, CostCtx);
  break;
```

then it will be the same as before.